### ISSUE TITLE
OSMAnd Live: Award to mappers must not be relative to changesets

### ISSUE THREAD

**Issue by vlotorev** [osmandapp/Osmand#4925]
It seems recipients are awarded relative to a number of changesets. Most of advanced mappers use JOSM and changeset can take an hour and can consist of hundreds of objects (think of mapping scores of buildings along the street). This mapper will accept the same award as mapper using OSMAnd/MAPS.ME and uploading single POI as a changeset. This doesn't seem to be honest.

**Comment by scaidermern** [comment]
I agree that just looking at the number of changesets is very subjective. However I think these awards will always stay subjective, no matter how you change their calculation.

Switching to the number of changed elements could be an option. However adding 3 shops is more useful as adding 3 benches in my opinion. Likewise, adding a maxspeed tag to a road is more useful than adding a width tag for most people, although the width of the road is more difficult to obtain.

There will always be ways to exploit this. For example by adding surface=asphalt to major roads which doesn't really create more value, at least in more developed countries where asphalt is the standard surface for major roads. Similarly, a mapper could just move lots of objects by a tiny amount to exploit the system.

Maybe the current award system could be made less exploitable by using a different approach. Rate each mapper by the number of changed objects. Then divide them into three equally large groups: Top contributors, medium contributors and casual contributors. All mappers of each group will receive the same award, for example the top contributor group will receive 60%, the medium contributors will receive 30% and the casual contributors will receive 10%. This way, individual contributors can still try to exploit the system but their award is capped since they can't receive more than valid top contributors.

Not sure how good this approach really is. I guess there are various scientific publications about approaches to make such award systems less exploitable.

**Comment by polarbearing** [comment]
See also #3351 and #3789

**Comment by hieu-van** [comment]
I'm also concerned that why OSM doesn't count a mapper's contribution using number of changes (like in Google Maps) instead of changesets? If OSM keeps using that way of counting contributions, people will likely thinks a mapper with a thousand-change changeset is less active than a mapper with lots of changesets that contain just a few changes (maybe I'm more like that latter one) 😄

**Comment by scaidermern** [comment]
> I'm also concerned that why OSM doesn't count a mapper's contribution using number of changes (like in Google Maps) instead of changesets?

Evaluating the performance of individual contributors is not part of OSM's goals. But various third-party tools generate statistics about individual contributors, for example http://hdyc.neis-one.org/

**Comment by vshcherb** [comment]
Replying to original request this system is not about honesty and about work. Contributing to OSM should be still fun and not work that's why we don't aim to have a "good salary" for work but we hope to share 1 Beer a month every active mapper. The biggest issues is the cheating when you don't make the changes at all.
The idea to achieve it is pretty simple if you did exactly many changes as top 10% of mappers you will receive exactly same amount as top 1 or top 3. Being JOSM / Maps.me contributor it is still pretty easy to achieve top 10% just being active. Again the distribution is also not fare but what the main goal for the mapper is just to be top 10% :) 100-200 changesets a month.

**Comment by cebderby** [comment]
It appears this payment calculation is currently being exploited by new user "maus (snekin)".  In 13 days (at time of writing) the user has managed to create 4743 changesets, editing only in regions/subregions with good subscription amounts at the time of editing (things can change during the month) and achieving rank 1 in every region touched with single changes per changeset and reports of repeated re-saving of the same objects (eg generating version 106 of a river...)   Specific comments have been raised at https://www.openstreetmap.org/changeset/65417871 and at a few other of the user's changesets - we'll see what reaction we get.  Can I ask that you review this particular case, and see if this constitutes misuse of the OSM Live payment approach, and if the payments should be made.  The wider issue of payment calculation remains, although as an encouraging 'nudge' to contribute it seems to work excellently.  Cebderby

**Comment by vshcherb** [comment]
We've established the contact with "maus (snekin)" and still couldn't understand what is the issue. The local communicty in OSM RU also didn't have any objections so far. So, we need to be clear whether it is good or bad. 
BTW there is no need to make 4743 changesets to be in top 1, only 120 per month is enough.

We're planning to change the way of awards currently we have enough information and we keep collecting it and testing it, so the calculation will be based on changes.
